src/components/payments/splits/splitsBalance.tsx

```diff
@@ -0,0 +1,71 @@
+import { PaymentAddressDto } from "@/app/api/payments/route";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { useSplitEarnings } from "@0xsplits/splits-sdk-react";
+import { Split, Wallet2 } from "lucide-react";
+import { useEffect, useState } from "react";
+
+interface SplitsBalanceProps {
+  projectId: string;
+  paymentAddress: PaymentAddressDto;
+}
+
+export function SplitsBalance({
+  projectId,
+  paymentAddress,
+}: SplitsBalanceProps) {
+  const { splitEarnings, isLoading, status, error } = useSplitEarnings(
+    parseInt(paymentAddress.chain_id),
+    paymentAddress.wallet_address,
+    true,
+    [paymentAddress.wallet_address],
+  );
+  const [balance, setBalance] = useState<string>("0.00");
+  useEffect(() => {
+    if (splitEarnings != null) {
+      const formattedAmount =
+        splitEarnings.activeBalances?.[
+          "0x0000000000000000000000000000000000000000"
```

Comment:
you absolutely sure this is on 0x00000 yeah? 
Maybe let's test on Base by sending a little bit of ETH to that address we created, just to be 200%
Doesn't make sense to me still that it would be sending it to 0 address on layer 2's, maybe that's bc you were checking on sepolia

Reply:
99% sure, I'll create on just in case